pkg/did/client.go

```diff
@@ -33,11 +33,11 @@ const (
 )
 
 type discovery interface {
-	GetEndpoints(domain string) ([]*endpoint.Endpoint, error)
+	GetEndpoints(domain string) ([]*endpoint.Endpoint, int, error)
 }
 
 type selection interface {
-	SelectEndpoints(endpoints []*endpoint.Endpoint) ([]*endpoint.Endpoint, error)
+	SelectEndpoints(endpoints []*endpoint.Endpoint, numStakeholders int) ([]*endpoint.Endpoint, error)
```

Comment:
numStakeholders shouldn't be an argument - it should be passed in when creating the service. The caller of the function doesn't necessarily know what the policy is.

Reply:
The policy is in the consortium config - this can be different for each consortium, even with the same client.

Reply:
Then there should be a new selection client instance for each consortium.